RecoTracker/LST/interface/LSTPixelSeedInput.h

```diff
@@ -0,0 +1,119 @@
+#ifndef RecoTracker_LST_LSTPixelSeedInput_h
+#define RecoTracker_LST_LSTPixelSeedInput_h
+
+#include <memory>
+#include <vector>
+
+class LSTPixelSeedInput {
+public:
+  LSTPixelSeedInput();
+  ~LSTPixelSeedInput();
+
+  LSTPixelSeedInput(LSTPixelSeedInput const&) = delete;
+  LSTPixelSeedInput& operator=(LSTPixelSeedInput const&) = delete;
+  LSTPixelSeedInput(LSTPixelSeedInput&&);
+  LSTPixelSeedInput& operator=(LSTPixelSeedInput&&);
+
+  void setLSTPixelSeedTraits(std::vector<float> see_px,
+                             std::vector<float> see_py,
+                             std::vector<float> see_pz,
+                             std::vector<float> see_dxy,
+                             std::vector<float> see_dz,
+                             std::vector<float> see_ptErr,
+                             std::vector<float> see_etaErr,
+                             std::vector<float> see_stateTrajGlbX,
+                             std::vector<float> see_stateTrajGlbY,
+                             std::vector<float> see_stateTrajGlbZ,
+                             std::vector<float> see_stateTrajGlbPx,
+                             std::vector<float> see_stateTrajGlbPy,
+                             std::vector<float> see_stateTrajGlbPz,
+                             std::vector<int> see_q,
+                             std::vector<unsigned int> see_algo,
+                             std::vector<std::vector<int>> see_hitIdx) {
+    see_px_ = see_px;
```

Comment:
```suggestion
                             std::vector<std::vector<int>> hitIdx) {
    px_ = px;
```

cleaner/non-repetitive names